# IS211_Assignment1

Not sure if exception handling is correct in part 1, but the code runs when all tests passed.
